examples/gke-a3ultragpu-staging.yaml

```diff
@@ -0,0 +1,181 @@
+# Copyright 2024 "Google LLC"
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+blueprint_name: gke-a3-ultra-staging
+
+vars:
+  project_id: hpc-toolkit-dev-staging
+  deployment_name: gke-a3-ultra-staging
+  region: us-east5
+  zone: us-east5-staginga
+  # Cidr block containing the IP of the machine calling terraform.
+  # The following line must be updated for this example to work.
+  authorized_cidr: 10.88.0.4/32
+
+deployment_groups:
+- group: primary
+  modules:
+  - id: a3ultra-gke-net-0
+    source: modules/network/vpc
+    settings:
+      network_name: a3ultra-gke-net-0
+      allowed_ssh_ip_ranges: [0.0.0.0/0]
+      mtu: 8996
+      subnetworks:
+      - subnet_name: a3ultra-gke-sub-0
+        subnet_region: $(vars.region)
+        subnet_ip: 192.168.0.0/18
+      secondary_ranges:
+        a3ultra-gke-sub-0:
+        - range_name: pods
+          ip_cidr_range: 10.4.0.0/14
+        - range_name: services
+          ip_cidr_range: 10.0.32.0/20
+      firewall_rules:
+      - name: a3ultra-gke-internal-0
+        ranges: [192.168.0.0/16]
+        allow:
+        - protocol: tcp
+          ports: ["0-65535"]
+        - protocol: udp
+          ports: ["0-65535"]
+        - protocol: icmp
+
+  - id: a3ultra-gke-net-1
```

Comment:
Are these 2 networks for a 2 node cluster ? And if so do we need 1 network for each node customer wants to add ?

Reply:
It's per the RDMA user guide (will share offline) and unrelated to the node count.

3 VPCs overall are required. 1 primary. 1 secondary. 1 RDMA VPC.

Reply:
Sounds good, we can add reference to the public guide once it is available.